lib/usb_connection.js

```diff
@@ -142,6 +128,15 @@ USB.Connection.prototype.open = function() {
         self.serialNumber = data;
         // Register this connection with daemon (keeps track of active remote processes)
         Daemon.register(self);
+
+        // If the USB Pipe isn't enabled on the other end (ie it is booting)
+        self.epIn.on('error', function(err) {
```

Comment:
This event will be thrown if Tessel isn't ready to open the USB pipe (ie if OpenWRT isn't booted) once `_receiveMessages` is called..